src/main/java/com/google/sps/data/TextAnalyser.java

```diff
@@ -159,6 +159,20 @@ private ClassifyTextResponse classify() throws IOException {
     return events;
   }
 
+  public Set<String> getGreetings() {
+    Set<String> greetings = new LinkedHashSet<String>();
+    String[] allGreetings = new String[] {"good morning", "congratulation", "welcome", "good evening", 
```

Comment:
Any reason to don't use an Enum class here?

Reply:
That's Mara's code which is already in master:)

Reply:
Golden rule while working in a team project:
Leave the campground cleaner than you found it

Also, if it's already in master it shouldn't mark it as new change. Please sync your work space before sending a PR